docs/jitserver_tuning.md

```diff
@@ -42,14 +42,16 @@ The client-session caches are deleted when the clients terminate, but this can h
 
  To enable this feature, specify the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) command line option, both at the server and at the client JVM.
 
- A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache named `default`.
+ A JITServer instance can have several AOT caches, each with its own name. This addresses the situation when client JVMs with significantly different profiles of execution use the same JITServer instance. A client JVM can indicate a specific AOT cache it wants to use by providing its name with the following command-line option [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename.md). If the client doesn't specify a name for the AOT cache, the server uses a cache named `default`.
 
  The maximum amount of memory that all the AOT cache instances combined can use at the server is 300 MB, by default. You can change this value by using the [`-XX:JITServerAOTmx=<size>`](xxjitserveraotmx.md) option. When the cache size reaches the specified limit, new clients cannot create new AOT cache instances or add new compiled methods to the existing AOT cache instances.
 
- Current limitations:
+ JITServer AOT cache can be reused by different instances of its JITServer server by enabling the AOT persistence feature with the [`-XX:+JITServerAOTCachePersistence`](xxjitserveraotcachepersistence.md) option. With this feature enabled, the JITServer server periodically saves its AOT caches to files. The server saves a new file when the number of extra AOT methods added to the in-memory cache since the last save equals or exceeds the number set in the `-Xjit:aotCachePersistenceMinDeltaMethods=<number of methods>` option. The default number is 200 methods.
+
```

Comment:
I find the description of periodic saving and the associated `-Xjit:` options a bit confusing. I think it could be rewritten in a simpler way to get across the point that a save operation is triggered once both conditions are satisfied: at least `-Xjit:aotCachePersistenceMinDeltaMethods=` (default 200) methods have been added, and at least `-Xjit:aotCachePersistenceMinPeriodMs=` (default 10000) milliseconds have passed since the previous save operation.

Reply:
How about this:

Typically, each JITServer server populates its own AOT caches independently of other existing servers, possibly
leading to duplicate work.
To help with JITServer autoscaling, and in particular with scaling down to 0, JITServer instances have the possibility
of saving their AOT caches to files. This feature is referred to as `AOT cache persistence` and can be enabled with
`-XX:+JITServerAOTCachePersistence` command line option. Subsequently launched JITServer instances can load
existing AOT cache files into their memory, and then continue to gradually augment them with new AOT compiled methods.
Saving an AOT cache to file is performed periodically, once the in-memory cache has added 200 extra AOT methods,
but no more frequent than every 10000 ms. These default tuning values can be configured with 
`-Xjit:aotCachePersistenceMinDeltaMethods=<N>` and `-Xjit:aotCachePersistenceMinPeriodMs=<N>` options respectively.